src/frontend/shared-components/src/loader/Loader.stories.js

```diff
@@ -7,12 +7,6 @@ export default {
   component: Loader
 };
 
-export const Default = () => (
-  <div style={{ background: "grey" }}>
-    <Loader />
-  </div>
-);
```

Comment:
Reason for removing the default?

Reply:
yeah dont need the grey background, the default is `Loader` 😄